pages/Documents/MessagingChannels/MobileAppMessagingSDKforAndroid/ReleaseNotes/4.9.0.md

```diff
@@ -51,7 +51,7 @@ DatePicker JSON schema is only supported on accounts using UMS version 4.2, plea
 
 # Known issue
 
-Deep links shared via Structured Content for InApp navigation are failing due to `http` prefix is getting added when clicked on a link. The workaround for this issue is to use [structured_content_link_as_callback](mobile-app-messaging-sdk-for-android-sdk-attributes-5-0-and-above.html#structured-content) configuration of SDK and handle links inside host app when fired SDK event:  [LP_ON_STRUCTURED_CONTENT_LINK_CLICKED](mobile-app-messaging-sdk-for-android-sdk-apis-callbacks-index.html#structured-content-link-clicked)
+Deep links shared via Structured Content for InApp navigation are failing due to the `http` prefix getting added when clicking on a link. The workaround for this issue is to use the [structured_content_link_as_callback](mobile-app-messaging-sdk-for-android-sdk-attributes-5-0-and-above.html#structured-content) configuration of SDK and handle links inside a host app when firing an SDK event:  [LP_ON_STRUCTURED_CONTENT_LINK_CLICKED](mobile-app-messaging-sdk-for-android-sdk-apis-callbacks-index.html#structured-content-link-clicked)
```

Comment:
Optional (appreciated but not required): Can you replace those double spaces by single spaces (after “SDK event:”)?

Affects all three files.